Structure_Engine/Query/TotalThickness.cs

```diff
@@ -198,6 +198,18 @@ public static double TotalThickness(this BuiltUpRibbed property)
             return property.TopThickness + property.RibHeight;
         }
 
+        /***************************************************/
+
+        [Description("Gets the total thickness of the SurfaceProperty.")]
+        [Input("property", "The property to evaluate.")]
+        [Output("TotalThickness", "The total thickness, including any ribs or waffling.", typeof(Length))]
```

Comment:
```suggestion
        [Output("totalThickness", "The total thickness, including any ribs or waffling.", typeof(Length))]
```

Can we do a find replace for the whole file, seems to occur on all the methods for `TotalThickness`.